src/components/github-htmlview.component.js

```diff
@@ -290,6 +297,48 @@ export class GithubHtmlView extends Component {
 
           return undefined;
         },
+        details: (node, index, siblings, parent, defaultRenderer) => {
+          const detailsChildren = node.children || [];
+          const childrendComponent = [];
+          let summaryText = 'details';
+
+          detailsChildren.forEach((child, childIdx) => {
+            if (child.type === 'tag' && child.name === 'summary') {
+              const summaryChild = child.children[0] || {};
```

Comment:
from experience, I know that using .children[x] is bad. Imagine if the first child is a space for example.
You may want to use the `onlyTagsChildren` here.

Reply:
Yeah, I'm not familiar with markdown, I basically keep switching to http://astexplorer.net/ to check the AST parsing result :p 
I didn't notice we may have space in first children 😭 

Should use `onlyTagsChildren` here. 👍